contribute/contributing_for_everyone.md

```diff
@@ -16,13 +16,13 @@ There is a spectrum of benefits to contributing - some more obvious than others.
 
 Types of contributions vary depending on your experience, background, and the nature of your interest in Julia. You can contribute in one or more of the following ways:
 
-- improve documentation,
-- [share a new package you created](https://github.com/elizavetasemenova/www.julialang.org/blob/main/contribute/creating_package.md),
+- improve documentation (this includes [fixing typos](#find-a-readme-file-to-edit)),
```

Comment:
We can add something like (this includes fixing typos and broken links)

Reply:
thanks, have added